docs/build/README.mdx

```diff
@@ -19,7 +19,7 @@ write both confidential and non-confidential smart contracts.
 For an overview of the other ParaTimes, refer to the [Other ParaTimes] chapter.
```

Comment:
Maybe add a new `## Sapphire` subtitle for this part for consistency with the toolbar and the rest of the chapter.